6_visualize/src/create_gif_tasks.R

```diff
@@ -1,9 +1,69 @@
-# create task table for gif generation
+# create task tables for gif generation - first table is the animation showing
+# how gages map to sparklines, second table shows the storm+precip+stage changes
+# over time
 
+create_intro_gif_tasks <- function(n_timesteps, folders){
 
+  # set up a series of animation frame timesteps (not related to actual time,
+  # just describe time within the explanatory animation)
+  timestep <- seq_len(n_timesteps)
 
-create_gif_tasks <- function(timestep_ind, folders, storm_cfg){
+  # aspect/resolution configuration placeholder. see same code in
+  # create_storm_gif_tasks - we should extract this into shared location once we
+  # get multiple configurations really going
+  cfgs <- c('a')
 
+  tasks <- tidyr::crossing(timestep, cfgs) %>%
+    unite(task_name, cfgs, timestep, sep = '_', remove = F) %>%
+    mutate(date_hour = sprintf('00_%03d', timestep),
```

Comment:
what is the significance of the "00_03" here?

Reply:
`00_` is just a prefix that will sort before the datestamps of the main fig. we may want to make this `10` or `01` to leave room for a t0 thumbnail gif in the future.

`%03d` is just telling sprintf to plot the integer value of timestep with a total of 3 digits, 0-padding as needed.